src/probnum/diffeq/steprule.py

```diff
@@ -1,16 +1,21 @@
 from abc import ABC, abstractmethod
 
+import numpy as np
+
 
 class StepRule(ABC):
     """(Adaptive) step size rules for ODE solvers."""
 
+    def __init__(self, firststep):
+        self.firststep = firststep
+
     @abstractmethod
-    def suggest(self, laststep, errorest, **kwargs):
+    def suggest(self, laststep, errorest, localconvrate=None):
```

Comment:
since solvers have orders, localconvrate cannot be an attribute of StepRule anymore, because it is initialised independently. In any case, now that we have it like this,I find it more convenient as an argument anyway.